x-pack/plugins/lens/public/editor_frame_service/service.tsx

```diff
@@ -68,10 +68,6 @@ export class EditorFrameService {
 
     const getStartServices = async () => {
       const [coreStart, deps] = await core.getStartServices();
-      const [resolvedDatasources, resolvedVisualizations] = await Promise.all([
-        collectAsyncDefinitions(this.datasources),
-        collectAsyncDefinitions(this.visualizations),
-      ]);
```

Comment:
I thought it was breaking our abstractions and would prefer to keep these hidden.